README.md

```diff
@@ -99,7 +101,11 @@ fi
 
 ![](assets/runscript.png)
 
-Alternatively, if you've installed SwiftLint via CocoaPods the script should look like this:
+You might want to move your SwiftLint phase directly before 'Compile Sources' step, to detect errors quickly before compiling.
```

Comment:
i'm not sure we should suggest this since SwiftLint usually assumes that you have valid Swift code, which means that if for some reason you have syntax errors, you might get non-sense warnings or, in the worst case, even crashes when running `swiftlint`

Reply:
Okay, I've never been aware of any best-practice in this area. We've placed it both before the compile step, and as the last step with equal results.

Would you prefer to have this paragraph but with a warning, or just remove it?